test/validations.test.js

```diff
@@ -558,7 +558,58 @@ describe('validations', function() {
   });
 
   describe('numericality', function() {
-    it('should validate numericality');
+    it('should succes if property is a number', function() {
+      User.validatesNumericalityOf('age');
+      var user = new User({age: 10});
+      user.isValid().should.be.true();
+    });
+    it('fails when given non-numeric values', function() {
```

Comment:
Please add one blank line between each test to make the code more readable... This applies for all your tests